docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: TraceQL
+menuTitle: TraceQL
+description: Learn about TraceQL, Tempo's query language for traces
+weight: 450
+aliases:
+  - /docs/tempo/latest/traceql/
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. Currently, TraceQL query can select traces based on the following see [future work](./architecture.md) for where the language is heading.
+
+- Span and resource attributes, timing, and duration
+- Basic aggregates: `count()` and `avg()`
+
+The TraceQL language uses similar syntax and semantics as [PromQL](https://grafana.com/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/) and [LogQL](https://grafana.com/docs/loki/latest/logql/), where possible. TraceQL recognizes two types of data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs.
```

Comment:
> TraceQL recognizes two types of data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs.

This sentence feels like it's in an awkward spot. Just remove?

Reply:
The sentence adds valuable information about attributes and intrinsics, which we have sections for below. I'll see about moving it to another section. Maybe it will fit better there.

Reply:
I moved the sentence down to the Construct a query section. What do you think?

Reply:
yeah, agree it makes more sense down there.